lib/THC/THCTensorIndex.cu

```diff
@@ -10,7 +10,6 @@
 #include <thrust/fill.h>
 #include <thrust/functional.h>
 #include <thrust/reduce.h>
```

Comment:
We don't seem to be using thrust at all in this file. Can you just remove all of the includes?